contents/tutorials/canary-release.md

```diff
@@ -0,0 +1,71 @@
+---
+title: How to do a canary release with feature flags in PostHog
+sidebar: Docs
+showTitle: true
+author: ['ian-vanagas']
+date: 2022-12-19
+featuredImage: ../images/tutorials/banners/canary-release.png
+topics: ["feature flags", "persons"]
+---
+
+Few things are worse for a developer than shipping a new feature, having it unexpectedly break, and frantically work to fix it. This can negatively impact your product, user experience, and engineering productivity. To mitigate problems with new features, teams often roll out features gradually to test for issues.
```

Comment:
```suggestion
Few things are worse for a developer than shipping a new feature, having it unexpectedly break, and then scrambling to fix it. This can negatively impact your product, user experience, and engineering productivity. To mitigate problems with new features, teams often roll out features gradually to test for issues.
```

It may be me over-thinking, but I think the tense gets a bit weird here. One way round it is: 'shipping a new feature, having it unexpectedly break, and having to frantically work to fix it.' - but that feels repetitive. Maybe cutting the adverb and using a new verb like this helps? Up to you.

Reply:
I like this better, I did think it was a bit awkward too.